src/components/icons/icons.visualroute.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
 import styled from '@emotion/styled';
+import { Switch, Route } from 'react-router-dom';
 import * as UIKit from 'ui-kit';
 import { Suite, Spec } from '../../../test/percy';
 
-const Inline = styled.div`
+const IconList = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
FYI in case you weren't aware, this could also be written as `repeat(8, 1fr)` 🤷‍♂️